kivy/core/gl/__init__.py

```diff
@@ -88,3 +88,11 @@ def print_gl_version():
     # To be able to use our GL provider, we must have a window
     # Automatically import window auto to ensure the default window creation
     import kivy.core.window  # NOQA
+
+
+def initialize_opengl():
+    from kivy.logger import Logger
+    Logger.critical('GL: Test failure')
+    msgbox('Test OpenGL failure')  # Force msgbox call
+
+initialize_opengl()
```

Comment:
Hi @Aakiboi,

Can you help me understand why this is needed for your use-case?